docs/webhooks/index.html

```diff
@@ -253,6 +254,36 @@ <h2>Events for CodeSignal Test</h2>
  <div class="subsection">
 <h3 id="testworkflow">Test webhooks process flow</h3>
 <img alt="Diagram of webhooks process flow for CodeSignal Test" width="100%" src="https://codesignal.com/wp-content/uploads/2021/03/webhooksproces_test.png" />
+</div>
+<div class="subsection">
+    <h3 id="companyTestSessionCreated">companyTestSessionCreated</h3>
+    <p>
+        Fired when a client creates a test session.
```

Comment:
I would rephrase this to say "Fired when a test session is created" -- it can be created in a numerous of ways, including automatic creation through the ATS or when a candidate follows a public URL.